inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -80,8 +88,29 @@ void FrontEnd::parseInterp(const Model& model, const ie::CNNLayerPtr& layer, con
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "Interp stage with name {} must have only 1 output, "
                      "actually provided {}", layer->name, outputs.size());
+    ie::details::CaselessEq<std::string> cmp;
+    const auto coord = layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
+    const auto interpMode = layer->GetParamAsString("mode", "linear");
```

Comment:
Do default values reflect specification?

Reply:
Yes. The modes are set to their default values in the Interpolate stage, not here.
This one (Interp) is only called in case when `mode == linear ||  mode == linear_onnx`